### PR TITLE
Add: Decimal Note Lane/Width, Note Properties Panel

### PR DESCRIPTION
**Pull request by KaleChips22**
Adds:
- Note Properties Panel:
    - Specific details per note
    - This is where you can set floating point values for not lane and width

Changes:
- Support for decimal note lane and width
- cyanvas version 6 in editor

Needed:
- Japanese translation

Issues:
closes: #10 
closes: #11

### CODE REVIEW

File: MikuMikuWorld/Note.cpp [hunk 1]
```diff
@@ -70,7 +70,7 @@ namespace MikuMikuWorld
 	int getFlickArrowSpriteIndex(const Note& note)
 	{
 		int startIndex = note.critical ? 24 : 12;
-		return startIndex + ((std::min(note.width, 6) - 1) * 2) +
+		return startIndex + ((std::min(note.width, (float)6) - 1) * 2) +
```
Comment (sevenc-nanashi) [inline MikuMikuWorld/Note.cpp:73]:
```suggestion
		return startIndex + ((std::min(note.width, 6.f) - 1) * 2) +
```

**Review by sevenc-nanashi — COMMENTED** [review]
tysm for pr! I haven't checked if this actually works (I was busy today), I'll do another review when I get home.
Also: I'll make beta release after this pr was merged.

**Comment by sevenc-nanashi** [comment]
fyi you should:
- Pull before PR. So commit log shown here will be more clean.
- separate branch for every PR. This prevents warning on GitHub (You can google "How to update my fork from command line" etc)

**Comment by mikreo** [comment]
I think right now we need to add eases, step type and hold type to the menu.
It'll probably also be good to clamp tap notes to more than 0 width. Holds don't need this, but it could automatically set the hold type to hidden as well.

**Comment by sevenc-nanashi** [comment]
I'm sure that I cannot find all bugs rn, so I'll merge and make prerelease.

**Pull request merged**